### PR TITLE
Pass multiple entities

### PR DESCRIPTION
**Pull request by Michlipski**
Allows users to pass an array of options for different entities to retrieve from the API

### CODE REVIEW

**Comment by thinhle-agilityio** [comment]
@Michlipski thanks for your contribution, really appreciate it. I have some comments as well

**Comment by Michlipski** [comment]
@thinhle-agilityio Just let me know if anything needs fixing up and I will address it 👍

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "crypto": "^1.0.1",
     "deep-map-keys": "^1.2.0",
     "json-stringify-safe": "^5.0.1",
+    "lodash": "^4.17.11",
```
[the remark this answers is not in the document]
Comment (thinhle-agilityio) [reply package.json]: @Michlipski pls take a look

File: src/fetch.js [hunk 2]
```diff
@@ -6,17 +6,16 @@ const chalk = require('chalk')
...
-  headers,
-  data,
-  name,
-  localSave,
-  path,
-  payloadKey,
-  auth,
-  auth0Config,
-  params,
+  _url,
```
[the remark this answers is not in the document]
Comment (thinhle-agilityio) [reply src/fetch.js]: @Michlipski

File: README.md [hunk 3]
```diff
@@ -33,6 +33,16 @@ plugins: [
   {
     resolve: 'gatsby-source-apiserver',
     options: {
+      // Pass an array containing any number of the entity configuration properties (except verbose, auth0Config),
+      // any not specified are defaulted to the general properties that are specified 
+      entitiesArray: [{
```
[the remark this answers is not in the document]
Comment (thinhle-agilityio) [reply README.md]: @Michlipski

**Comment by Michlipski** [comment]
@thinhle-agilityio this should address everything, let me know if I'm missing anything

**Pull request merged**